libs/designsystem/card/src/card.component.scss

```
@@ -57,6 +58,24 @@
     box-shadow: none;
   }
 
+  &.outlined {
+    background-color: transparent;
+    border: 1px solid utils.get-color('medium');
+    box-shadow: none;
+
+    &:has(kirby-card-header.flagged) {
+      border: none;
+
+      .content-wrapper {
+        border-block-end: 1px solid utils.get-color('medium');
+        border-inline-start: 1px solid utils.get-color('medium');
+        border-inline-end: 1px solid utils.get-color('medium');
```

I guess we should allow a custom border in outlined cards as well? 🤔 
```suggestion
        border-block-end: var(--kirby-card-border, 1px solid utils.get-color('medium'));
        border-inline-start: var(--kirby-card-border, 1px solid utils.get-color('medium'));
        border-inline-end: var(--kirby-card-border, 1px solid utils.get-color('medium'));
```